packages/apputils-extension/src/index.ts

```diff
@@ -581,6 +583,20 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
         }
       }
     });
+
+    commands.addCommand(CommandIDs.runAllCommands, {
+      label: trans.__('Run All Commands Passed as Args'),
+      execute: async (args) => {
+        const commandsString: string[] = args.commands as string[];
+        for (let i = 0; i < commandsString.length; i++) {
+          const cmd = commandsString[i];
```

Comment:
Maybe this fragment could be simplified by using [for..of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)?

```suggestion
        for (const cmd of commandsString) {
```